src/main/resources/cpp-modern-json/struct-header.mustache

```diff
@@ -23,6 +23,8 @@ namespace {{this}} {
 {{/description}}
 struct {{classname}}{{#parent}} : public {{{parent}}}{{/parent}} {
 
+    virtual ~{{classname}}() = default;
```

Comment:
Hm, I though we only need the virtual d'tor in the parent class. Moreover, this would add it to any struct, not just the ones that inherit, wouldn't it?

Reply:
That would mean you just `{{#hasChildren}}` this line, right?

Reply:
yep